platform.md

```diff
@@ -148,6 +159,441 @@ A platform MAY support any number of mixins for a given stack in order to suppor
+    - MUST contain all buildpack-provided launch layers as determined by the [Buildpack Interface Specfication](buildpack.md)
+    - MUST contain one or more app layers as determined by the [Buildpack Interface Specfication](buildpack.md)
+    - MUST contain a layer that includes `<launcher>`
+    - MUST contain a layer that includes `<layers>/config/metadata.toml`
+    - MUST have `ENTRYPOINT=<launcher>`
+    - MUST contain the following `Env` entries
+      - `"CNB_LAYERS_DIR=<layers>"`
+      - `"CNB_APP_DIR=<app>"`
+    - MUST contain the following `Env` entry, IF `<process-type>` is set
+      - `"CNB_PROCESS_TYPE=<process-type>"`
+    - MUST contain the following labels
+        - `io.buildpacks.lifecycle.metadata`: see [lifecycle metadata label (JSON)](#lifecycle-metadata-label-(json))
+        - `io.buildpacks.project.metadata`: the value of which SHALL be the json representation `<project-metadata>`
+        - `io.buildpacks.build.metadata`: see [build metadata (JSON)](#build-metadata-label-(json))
+
+IF a cache is provided the lifecycle:
+- SHALL write the contents of all cached layers to the cache
+- SHALL record the DiffID and layer content metadata of all cached layers in the cache
+
+#### `creator`
+The platform MUST execute `creator` in the **build environment**
+
+Usage:
+```
+/cnb/lifecycle/creator <image>
+  [-app <app>] \
+  [-cache-dir <cache-dir>] \
+  [-cache-image <cache-image>] \
+  [-daemon] \ # sets <daemon>
+  [-gid <gid>] \
+  [-launch-cache <launch-cache> ] \
+  [-launcher <launcher> ] \
+  [-layers <layers>] \
+  [-log-level <log-level>] \
+  [-order <order>] \
+  [-platform <platform>] \
+  [-previous-image <previous-image> ] \
+  [-process-type <process-type> ] \
+  [-project-metadata <project-metadata> ] \
+  [-run-image <run-image>] \
+  [-skip-restore <skip-restore>] \
+  [-stack <stack>] \
+  [-tag <tag>...] \
+  [-uid <uid> ]
+```
+
+Running `creator` SHALL be equivalent to running `detector`, `analzyer`, `restorer`, `builder` and `exporter` in order with identical inputs where they are accepted, with the following exceptions.
+
+| Input             | Environment Variable| Default Value| Description
+|-------------------|---------------------|--------------|----------------------
+| `<previous-image>`| `CNB_PREVIOUS_IMAGE`| `<image>`    | Image reference to be analyzed (usually the result of the previous build)
+| `<skip-restore>`  | `CNB_SKIP_RESTORE`  | `false`      | Do not write layer metadata or restore cached layers
+| `<tag>...`        |                     |              | Additional tag to apply to exported image
+
+- IF `<skip-restore>` is `true` the `creator` SHALL skip layer analysis and skip the entire Restore phase.
+- IF the platform provides one or more `<tag>` inputs they SHALL be treated as additional `<image>` inputs to the `exporter`
+
+Outputs produced by `creator` are identical to those produced by `exporter`.
```

Comment:
Maybe it's worth adding why a user would want to use the creator over the 5 phases separately.

Reply:
I agree that this would be useful, but I wonder if that's just the difference b/t spec and documentation?